apps/wpcom-block-editor/src/wpcom/features/tracking/wpcom-block-editor-save-click.js

```diff
@@ -12,14 +12,18 @@ export const wpcomBlockEditorSaveClick = () => ( {
 	selector:
 		'.editor-entities-saved-states__save-button, .editor-post-publish-button:not(.has-changes-dot)',
 	type: 'click',
-	handler: () => {
+	handler: ( event ) => {
 		const isSiteEditor = getEditorType() === 'site';
 		const isCurrentPostPublished = select( 'core/editor' ).isCurrentPostPublished();
 		const isEditedPostBeingScheduled = select( 'core/editor' ).isEditedPostBeingScheduled();
 		let actionType = 'publish';
 
 		if ( isSiteEditor ) {
-			actionType = 'save';
+			if ( event.target.textContent.includes( 'Activate' ) ) {
```

Comment:
We need to factor in different translations so we can't depend on the text being in English.

Reply:
I had a try to address this 🙏 https://github.com/Automattic/wp-calypso/pull/81000/commits/a7870dde524001f07c05fca8a5dd0d902e52a8b9